### PR TITLE
T.herman/scene preprocessing

### PR DESCRIPTION
**Pull request by tomasz-herman**
RenderedScene

### CODE REVIEW

File: RayTracer/Source/World/Scene.cs [hunk 1]
```diff
@@ -9,6 +9,7 @@ namespace RayTracing.World
     public class Scene : IHittable
     {
         public List<Model> Models { get; } = new List<Model>();
+        public List<IHittable> Hittables { get; } = new List<IHittable>();
```
Comment (karolk98) [inline RayTracer/Source/World/Scene.cs:12]: We should decide on one or the other IMO

Comment (karolk98) [reply RayTracer/Source/World/Scene.cs:12]: I mean Models or Hittables

File: RayTracer/Source/World/Scene.cs [hunk 2]
```diff
@@ -28,9 +29,9 @@ public bool HitTest(Ray ray, ref HitInfo hit, float from, float to)
             bool hitAnything = false;
             float closest = to;
 
-            foreach (var model in Models)
+            foreach (var hittable in Hittables)
```
Comment (karolk98) [inline RayTracer/Source/World/Scene.cs:32]: Why do we even do this?

File: RayTracer/Source/World/Scene.cs (hunk 1 above)
Comment (karolk98) [reply RayTracer/Source/World/Scene.cs:12]: Or at least make it private

Comment (karolk98) [reply RayTracer/Source/World/Scene.cs:12]: both of those

**Review by karolk98 — APPROVED** [review]

**Review by p-flis — APPROVED** [review]

**Pull request merged**